packages/ui/src/2_molecules/Dialog/components/DialogBody.module.css

```diff
@@ -0,0 +1,3 @@
+.container {
```

Comment:
You don't need this file at all, just apply the classes directly. Also, I don't think you need to set text color in dialog body as the only place where it needs to be set is dialog header.